exec/java-exec/src/main/java/org/apache/drill/exec/record/BatchSchema.java

```diff
@@ -162,20 +162,22 @@ private boolean majorTypeEqual(MajorType t1, MajorType t2) {
    * Merge two schema to produce a new, merged schema. The caller is responsible
    * for ensuring that column names are unique. The order of the fields in the
    * new schema is the same as that of this schema, with the other schema's fields
-   * appended in the order defined in the other schema. The resulting selection
-   * vector mode is the same as this schema. (That is, this schema is assumed to
-   * be the main part of the batch, possibly with a selection vector, with the
-   * other schema representing additional, new columns.)
+   * appended in the order defined in the other schema.
+   * <p>
+   * Merging data with selection vectors is unlikely to be useful, or work well.
```

Comment:
Can you please leave a comment about why this is unlikely to be useful, or work well?